pkg/ring/kv/client.go

```diff
@@ -67,7 +67,7 @@ func (cfg *Config) RegisterFlagsWithPrefix(flagsPrefix, defaultPrefix string, f
 // It also deals with serialisation by using a Codec and having a instance of
 // the the desired type passed in to methods ala json.Unmarshal.
 type Client interface {
-	// Get a spefic key.  Will use a codec to deserialise key to appropriate type.
+	// Get a specific key.  Will use a codec to deserialize key to appropriate type.
```

Comment:
Now that I've learned to write 'serialise' everywhere, you tell me it's wrong? :-)

Seriously though, one seems to be British English spelling, while the other one is American English. Neither is good or bad.

Reply:
Thanks. I will change it back